library/src/main/java/com/airbnb/android/airmapview/AirMapType.java

```diff
@@ -10,9 +10,9 @@
  */
 public class AirMapType {
 
-  private static final String ARG_MAP_DOMAIN = "map_domain";
-  private static final String ARG_FILE_NAME = "map_file_name";
-  private static final String ARG_MAP_URL = "map_url";
+  protected static final String ARG_MAP_DOMAIN = "map_domain";
+  protected static final String ARG_FILE_NAME = "map_file_name";
```

Comment:
Yep, originally I had to while working out how to get `AirMapType` to play with `MapboxWebMapType`.  9e7eebd9b95e20e4bc90378e505b7961c79fe4f8 was the first step in cleaning that up.  I can and will switch this back to `private` now.